playground/DatabaseMigration/DatabaseMigration.ApiModel/Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+
+  <PropertyGroup>
+    <!-- Clear solution default because EF tooling doesn't like a custom value. -->
```

Comment:
Can you file a bug with EF for this?

Reply:
It's tracked by https://github.com/dotnet/efcore/issues/23853